### PR TITLE
#7750 Create automated test to verify map functionality for Event

### LINKED ISSUE #7750: Create automated test to verify map functionality for Event
The aim of the test is to check the map functionality. 
In order to verify the component, please create an automated test based on the following logic;

1. Create new event via API
2. Navigate to event based on UUID
3. Assert that map icons are disabled by default (for all address empty fields)
4. Fill all field excepting latitude and longitude
5. Assert map icon is disabled
6. Fill latitude
7. Assert map icon is disabled
8. Fill longitude
9. Assert map icon is enabled
10. Click on map icon
11. Assert map popup is displayed

### PR DESCRIPTION
**Pull request by popadriangeo**
Fixes #7750 
Added Webhelper Methods
Added test steps
Added locator and methods for Edit Events Page(used in new test)
https://jenkins.sormas.netzlink.com/view/SORMAS-E2eTests/job/sormas-RunE2eTests-branch/1653/allure/#suites/ba64af4344a5d1b16948f67eae90e934/351642695a369dc2/

https://jira.vitagroup.ag/browse/SOR-3628

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java [hunk 1]
```diff
@@ -965,6 +965,20 @@ public void isElementGreyedOut(By elementLocator) {
...
+            Assert.assertTrue(
+                getAttributeFromWebElement(elementLocator, "class").contains("disabled")));
+  }
+
+  public void assertElementIsEnabled(By elementLocator) {
+    assertHelpers.assertWithPoll20Second(
+        () ->
+            Assert.assertFalse(
+                getAttributeFromWebElement(elementLocator, "class").contains("disabled")));
+  }
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java:980]: we already have a method in place checking this: isElementEnabled
Is it possible to use it and maybe if is not working at that locator level, to find one where this one works?

If not possible, create a single method to check this, add it next to this existing one to make sure people will see both, and maybe name it like isElementEnabledAtAttributeLevel

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java [hunk 2]
```diff
@@ -1787,6 +1792,53 @@ public EditEventSteps(
...
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EYE_ICON);
+          webDriverHelpers.assertElementIsDisabled(EYE_ICON);
+        });
+
+    Then(
+        "^I Verify The Eye Icon opening the Map is enabled in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EYE_ICON);
+          webDriverHelpers.assertElementIsEnabled(EYE_ICON);
+        });
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java:1807]: compress these 2 methods under a single one and make the enabled/disabled value as parameter

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java [hunk 3]
```diff
@@ -1787,6 +1792,53 @@ public EditEventSteps(
...
+          webDriverHelpers.submitInWebElement(GPS_LATITUDE_INPUT_EDIT_EVENT);
+        });
+
+    And(
+        "^I Add the GPS Longitude Values in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(GPS_LONGITUDE_INPUT_EDIT_EVENT);
+          webDriverHelpers.fillInWebElement(GPS_LONGITUDE_INPUT_EDIT_EVENT, "23.34555677");
+          webDriverHelpers.submitInWebElement(GPS_LONGITUDE_INPUT_EDIT_EVENT);
+        });
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java:1823]: make a single method and pass both parameters as variables, we shouldn't hardcode them

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature [hunk 4]
```diff
@@ -1278,4 +1278,18 @@ Feature: Create events
     Then I check if Present condition of person combobox has value "Alive"
     And I check if Present condition of person combobox has value "Dead"
     And I check if Present condition of person combobox has value "Unknown"
-    Then I check if Present condition of person combobox has no value "Buried"
\ No newline at end of file
+    Then I check if Present condition of person combobox has no value "Buried"
+
+  @env_main @#7750
+  Scenario: Check the map functionality
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature:1284]: specify also from which page

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java (hunk 1 above)
Comment (popadriangeo) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java]: isElementEnabled returns true for everything but disabled input elements.
Resolved by creating new method, isElementEnabledAtAttributeLevel

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**